### PR TITLE
fix build failures

### PR DESCRIPTION
**Pull request by whyrusleeping**
build was failing, both of the c++ code and in the go code.

### CODE REVIEW

**Comment by whyrusleeping** [comment]
although, the makefile change probably isnt necessary as i didnt need to build the .so, just the .a

File: udt/fd.go [hunk 1]
```diff
@@ -317,11 +322,11 @@ func closeSocket(sock C.UDTSOCKET) error {
 func dialFD(laddr, raddr *UDTAddr) (*udtFD, error) {
 
 	if raddr == nil {
-		return nil, &net.OpError{"dial", "udt", raddr, errors.New("invalid remote address")}
+		return nil, &net.OpError{Op: "dial", Net: "udt", Addr: raddr, Err: errors.New("invalid remote address")}
```
Comment (jbenet) [inline udt/fd.go]: is this a go 1.5 thing?

Comment (whyrusleeping) [inline udt/fd.go]: not necessarily, but at some point since we last built this, another field was added to the struct.

Comment (jbenet) [inline udt/fd.go]: ah

**Pull request merged**